### PR TITLE
Backport notes to changelog

### PR DESCRIPTION
**Pull request by amyblais**
For https://github.com/mattermost/mattermost-webapp/pull/7685, https://github.com/mattermost/mattermost-server/pull/17187, and https://mattermost.atlassian.net/browse/MM-34158

### CODE REVIEW

**Review by lieut-data — APPROVED** [review]

File: source/administration/changelog.md [hunk 1]
```diff
@@ -6,6 +6,8 @@ Also see [changelog in progress](https://bit.ly/2nK3cVf) for the next release.
 
 ## Release v5.33 - [Feature Release](https://docs.mattermost.com/administration/release-definitions.html#feature-release)
 
+- **v5.33.2, release day TBD**
+  - Fixing performance issues with the new sidebar that affects typing performance on busy servers. [MM-30407](https://mattermost.atlassian.net/browse/MM-30407)
```
Comment (lieut-data) [inline source/administration/changelog.md:10]: My only concern with the term `fixing` is that it implies a complete resolution, whereas I think there is still substantial improvements to be made.

0/5, but I wonder if we should phrase as:
```suggestion
  - Improving typing performance on busy servers with the new sidebar enabled. [MM-30407](https://mattermost.atlassian.net/browse/MM-30407)
```

File: source/administration/changelog.md [hunk 2]
```diff
@@ -6,6 +6,8 @@ Also see [changelog in progress](https://bit.ly/2nK3cVf) for the next release.
 
 ## Release v5.33 - [Feature Release](https://docs.mattermost.com/administration/release-definitions.html#feature-release)
 
+- **v5.33.2, release day TBD**
+  - Improving typing performance on busy servers with the new sidebar enabled. [MM-30407](https://mattermost.atlassian.net/browse/MM-30407)
```
Comment (lieut-data) [inline source/administration/changelog.md:10]: Oops, I really messed up the grammar here! Sorry:
```suggestion
  - Improved typing performance on busy servers with the new sidebar enabled. [MM-30407](https://mattermost.atlassian.net/browse/MM-30407)
```

File: source/administration/changelog.md [hunk 3]
```diff
@@ -254,6 +256,9 @@ Multiple setting options were added to ``config.json``. Below is a list of the a
 
 ## Release v5.31 - [ESR](https://docs.mattermost.com/administration/extended-support-release.html)
 
+- **v5.31.2, release day TBD**
+  - Improving typing performance on busy servers with the new sidebar enabled. [MM-30407](https://mattermost.atlassian.net/browse/MM-30407)
```
Comment (lieut-data) [inline source/administration/changelog.md:260]:
```suggestion
  - Improved typing performance on busy servers with the new sidebar enabled. [MM-30407](https://mattermost.atlassian.net/browse/MM-30407)
```

**Review by lieut-data — APPROVED** [review]

File: source/administration/changelog.md (hunk 2 above)
Comment (amyblais) [reply source/administration/changelog.md:11]: This can be changed once the dot release has been shipped.

**Pull request merged**